csrankings.csv

```diff
@@ -8956,8 +8964,8 @@ Jon Barker,University of Sheffield,https://www.sheffield.ac.uk/dcs/people/academ
 Jon Chamberlain,University of Essex,https://www.essex.ac.uk/people/chamb94408/jon-chamberlain,JJN-i1cAAAAJ
 Jon Crowcroft,University of Cambridge,https://www.cl.cam.ac.uk/~jac22,qnMs-XYAAAAJ
 Jon Doyle,North Carolina State University,https://www.csc.ncsu.edu/faculty/doyle,C6DtGmMAAAAJ
-Jon E. Froehlich,University of Washington,http://www.cs.umd.edu/~jonf,nExKrpsAAAAJ
-Jon Froehlich,University of Washington,http://www.cs.umd.edu/~jonf,nExKrpsAAAAJ
+Jon E. Froehlich,University of Maryland - College Park,http://www.cs.umd.edu/~jonf,nExKrpsAAAAJ

```

Comment:
I spot checked this listing and Froehlich's web page says he has been at UW since 2017.

Reply:
You are correct. I missed the recent affiliation of Dr. Froehlich. I have corrected it in the csrankings.csv file.